services/src/error.rs

```diff
@@ -399,3 +402,9 @@ impl From<proj::ProjError> for Error {
         Self::Proj { source }
     }
 }
+
+impl From<geoengine_operators::executor::error::ExecutorError> for Error {
+    fn from(source: geoengine_operators::executor::error::ExecutorError) -> Self {
+        Self::Executor { source }
+    }
+}
```

Comment:
You can put
`#[snafu(implicit)]`
on top of 
```
Executor {
        source: geoengine_operators::executor::error::ExecutorError,
    },
```
instead of implementing `From` yourself.

Reply:
Resolved here: 30a6c58dfaebf7bfc8c5bb4501ed48684c9a64a8